File: src/main/java/darkside/AbstractOrc.java
package darkside;

import metaside.MiddleEarthCitizen;

/**
 * Created by Petr on 15.10.2016.
 */
public abstract class AbstractOrc extends MiddleEarthCitizen {
    public AbstractOrc() {
        super(8, 10);
    }

    protected AbstractOrc(int minPower, int maxPower){
        super(minPower, maxPower);
    }
}

File: CSharpVersion/MiddleEarth/MiddleEarth/darkside/AbstractOrc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MiddleEarth.darkside
{
    abstract class AbstractOrc : metaside.MiddleEarthCitizen
    {
        public AbstractOrc(string name) : base(name, 8, 10){}

        protected AbstractOrc(string name, int minPower, int maxPower) : base(name, minPower, maxPower) { }
    }
}
